packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx

```diff
@@ -90,9 +92,19 @@ const InvoiceRow = (props: Props) => {
   )
   const {id: invoiceId, amountDue, creditCard, endAt, paidAt, payUrl, status} = invoice
   const isEstimate = status === 'UPCOMING'
+  const history = useHistory()
+
+  const goToInvoice = () => {
+    if (payUrl) {
+      window.open(payUrl, '_blank', 'noreferrer')
+    } else {
+      history.push(`/invoice/${invoiceId}`)
+    }
+  }
+
   return (
     <Row>
-      <RowLink rel='noopener noreferrer' target='_blank' to={`/invoice/${invoiceId}`}>
+      <RowLink onClick={goToInvoice}>
```

Comment:
-1 Previously also internal invoices would open in a new window. I think we could just do a
```ts
      <RowLink rel='noopener noreferrer' target='_blank' to={payUrl || `/invoice/${invoiceId}`}>
```
here